lib/discordrb/api/channel.rb

```diff
@@ -201,7 +201,7 @@ def delete_user_reaction(token, channel_id, message_id, emoji, user_id)
   # https://discord.com/developers/docs/resources/channel#get-reactions
   def get_reactions(token, channel_id, message_id, emoji, before_id, after_id, limit = 100)
     emoji = URI.encode_www_form_component(emoji) unless emoji.ascii_only?
-    query_string = "limit=#{limit}#{"&before=#{before_id}" if before_id}#{"&after=#{after_id}" if after_id}"
+    query_string = URI.encode_www_form({ limit: limit || 100, before: before_id, after: after_id }.compact)
```

Comment:
Could we possibly drop the `|| 100` here? It's redundant with the default value from the method signature.

Reply:
From the original bug report #38, `Message#reacted_with` will pass nil to the paginator (and therefore the API method) to get all results. The default return amount from the API is 25, so specifying 100 (aka max) will reduce the amount of API requests we need to make to get all results.